st2actions/st2actions/runners/actionchainrunner.py

```diff
@@ -130,13 +130,16 @@ def pre_run(self):
             raise runnerexceptions.ActionRunnerPreRunError(message)
 
     def run(self, action_parameters):
-
         result = {'tasks': []}  # holds final result we store
         context_result = {}  # holds result which is used for the template context purposes
         top_level_error = None  # stores a reference to a top level error
         fail = True
         action_node = None
 
+        # Add action parameters to the context result so they can be accessed
+        # inside "publish"
+        context_result['parameters'] = action_parameters
```

Comment:
Yep, I saw the results and I was wondering how and if we should prefix it.

I'm personally not a big fan of double underscore since `__` in Python means something internal which you shouldn't really touch. In the action parameters case, I want to prefix it to avoid potential name clashes and not because it's an "internal" / "secret" parameter.